test/document.shim.ts

```diff
@@ -1,32 +1,25 @@
 // eslint-disable-next-line import/no-mutable-exports
 let documentShim: any;
 
-try {
-  documentShim = document || {
-    head: {
-      appendChild: () => false,
+const shim = {
+  head: {
+    appendChild: (child: { onload?: () => void }) => {
+      child.onload?.();
```

Comment:
`documentShim.head.appendChild` seems to be the best place to forge the iframe `onload`. I removed some code from tests that was intended to do this